### PR TITLE
fix(DB/Locales): Add Missing esES/esMX locales to WG and GrizzlyHills

### PR DESCRIPTION
**Pull request by Valdifer**
## Changes Proposed:
-  Some quests from WG and GrizzlyHills were showing texts in english, adding those Locales translations
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- Executed query without errors
- 

## How to Test the Changes:

1. (13153, 13154, 13156, 13185, 13186, 13191, 13192, 13193, 13194, 13195, 13196, 13197, 13198, 13199, 13200, 13201, 13202, 13538, 13539, 12026, 12037, 12054, 12058, 12073, 12074, 12165, 12170, 12175, 12176, 12177, 12178, 12195, 12196, 12197, 12198, 12199, 12201, 12202, 12203, 12204, 12207, 12208, 12213, 12229, 12231, 12236, 12238, 12241, 12242, 12256, 12257, 12259, 12270, 12280, 12284, 12288, 12315, 12317, 12324, 12412, 12413, 12415, 12422, 12423, 12424, 12425, 12427, 12428, 12429, 12430, 12431, 12433, 12436, 12451, 12453, 12468, 12763, 13129)
2. .quest start (Select one of above)
3. Go and talk to the NPC before complete the quest
4. Take a look to the objective
5. .quest complete (Select one of above)
6. .Talk to the NPC

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1668938810265176700.sql [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Alianza eliminada en Bahía Ventura', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12317;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Rezagados de Ventura y Cía. ahuyentados', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12324;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Caballos mesteños de las Tierras Altas espantados', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12415;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Vista de águila imperial unida', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 12453;
+UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Huye de Arroyoplata', `ObjectiveText2` = '', `ObjectiveText3` = '', `ObjectiveText4` = '' WHERE `locale` IN ('esES', 'esMX') AND ID = 13524;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1668938810265176700.sql:237]: Missing new line at the end of the file.

**Review by Nefertumm — CHANGES_REQUESTED** [review]
Tested a few ones, all good, but merge both files into one and add the missing line at the end of the file

File: data/sql/updates/pending_db_world/rev_1668938810265176700.sql (hunk 1 above)
Comment (Valdifer) [reply data/sql/updates/pending_db_world/rev_1668938810265176700.sql]: > Missing new line at the end of the file.

Sry, i hope all is ok now

**Review by Nefertumm — APPROVED** [review]

**Comment by Nefertumm** [comment]
All good, thank you for your PRs!

**Pull request merged**